packages/cw1155/src/event.rs

```diff
@@ -0,0 +1,41 @@
+use cosmwasm_std::{attr, Response, Uint128};
+use cw_utils::Event;
```

Comment:
Could you try removing the dependency on this Event trait? I think we should remove it from cw_utils (https://github.com/CosmWasm/cw-utils/issues/17).

You probably can design this simpler without a trait but if you really need/want it, it's better to have it in here.

Thanks!